docs/src/api/params.md

```diff
@@ -531,15 +531,18 @@ Does not enforce fixed viewport, allows resizing window in the headed mode.
 - `clientCertificates` <[Array]<[Object]>>
   - `origin` <[string]> Exact origin that the certificate is valid for. Origin includes `https` protocol, a hostname and optionally a port.
   - `certPath` ?<[path]> Path to the file with the certificate in PEM format.
+  - `cert` ?<[string]> Direct value of the certificate in PEM format.
```

Comment:
Unsure if we should do `[Buffer]` everywhere or `[Buffer]|[string]` for a better user experience. Doing `Buffer` only seems the simplest to avoid overloads in e.g. .NET.

Reply:
I think buffer everywhere is good.

Reply:
Done.